extensions/wikidata/module/scripts/dialogs/perform-edits-dialog.js

```diff
@@ -30,34 +57,18 @@ PerformEditsDialog.launch = function(logged_in_username, max_severity) {
      dismiss();
   });
 
-  var hiddenIframe = $('#hiddenIframe').contents();
+  this._elmts.editSummary.keypress(function (evt) {
+    evt.preventDefault();
```

Comment:
you should not call `evt.preventDefault()` here since it blocks all key presses in the form. Test your PR: try adding a summary in the field, your key presses will not be recorded.